packages/stream/solana/StreamClient.ts

```diff
@@ -876,20 +1073,29 @@ export class SolanaStreamClient extends BaseStreamClient {
       );
       accounts = [...outgoingAccounts, ...incomingAccounts];
     } else {
-      const offset = direction === "outgoing" ? STREAM_STRUCT_OFFSET_SENDER : STREAM_STRUCT_OFFSET_RECIPIENT;
+      const offset =
+        direction === StreamDirection.Outgoing ? STREAM_STRUCT_OFFSET_SENDER : STREAM_STRUCT_OFFSET_RECIPIENT;
       accounts = await getProgramAccounts(this.connection, publicKey, offset, this.programId);
     }
 
     let streams: Record<string, Contract> = {};
 
+    if ((type === StreamType.All || type === StreamType.Vesting) && direction !== StreamDirection.Incoming) {
```

Comment:
why Incoming couldn't be aligned?

Reply:
cause they are fetched by 'recipient' field in the vesting contracts in that case. The `getOutgoingAligned` function needs to fetch the Streams through the proxy contracts as the Aligned Stream contracts have the `sender` field as the proxy PDA, and not the `address` (user wallet) being passed here.

@Yolley can confirm this, just as a sanity check for myself